src/svgeditor/ImageEdit.as

```diff
@@ -767,12 +768,14 @@ public class ImageEdit extends Sprite {
 			setToolMode("select");
 		}
 		// If the tool wasn't canceled and an object was created then select it
-		if (nextObject && nextObject.width > 0 && nextObject.height > 0 && (nextObject is Selection || nextObject.parent)) {
```

Comment:
Your new code doesn't include `|| nextObject.parent`... do you know why that was in the old code?